README.md

```diff
@@ -40,7 +40,7 @@ Down with YAML!
 Configure Tilt with a `Tiltfile`, written in a small subset of Python called
 [Starlark](https://github.com/bazelbuild/starlark#tour).
 
-To get started, check out some [examples](https://docs.tilt.build/first_example.html) or dive into the
+To get started, check out some [examples](https://docs.tilt.build/example_projects.html) or dive into the
```

Comment:
I'd push them towards the tutorial, or just the docs page in general.

Also, let's remove "Down with YAML!", because that makes it sound like want them to throw out their yaml.